src/components/CurrentDirectors/CurrentDirectors.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <div id="current-directors">
+
+    <v-card flat>
+      <!-- Header -->
+      <div class="header-container">
+        <v-icon color="app-dk-blue">mdi-account-multiple-plus</v-icon>
+        <label class="font-weight-bold pl-2">Current Directors</label>
+      </div>
+
+      <!-- Instructional Text -->
+      <div class="instructional-text pt-10 pb-15 px-4">
+        To change directors, please use the Change feature in the Current Directors list on you business dashboard.
+      </div>
+
+      <div class="list-container px-4">
+        <!-- List Display Section -->
+        <div id="people-roles-list" v-if="currentDirectors.length > 0">
+          <!-- List Headers -->
+          <v-row class="people-roles-list-header list-item__subtitle pb-3" no-gutters>
+            <v-col class="pr-5" v-for="(title, index) in tableHeaders" :key="index">
+              <span class="directors-title">{{ title }}</span>
+            </v-col>
+          </v-row>
+
+          <!-- List Content -->
+          <v-row
+            class="people-roles-content py-3"
+            v-for="(orgPerson, index) in currentDirectors"
+            :key="index"
```

Comment:
Not 100% but this key could create some duplicate identifiers if there is another mounted instance of `:key=index`.
An option to preserve the uniqueness would be template string a descriptor prefix to the index, like `director:${index}` or something along those lines.

Reply:
This looks like the only v-for on this page so there's no identifier conflict.

However, I agree with Cam that it's a good idea to specify a key value that is more than just the index (see Cam's example).

This is not an issue atm so I'm resolving this conversation.